core/registration.c

```diff
@@ -295,11 +326,40 @@ static int prv_update_registration(lwm2m_context_t * contextP, lwm2m_server_t *
     return 0;
 }
 
+void handle_registration_response(lwm2m_context_t * contextP, void * fromSessionH, coap_packet_t * message)
+{
```

Comment:
Yes and even more general, the “transaction” should support the separate responses pattern in general. Currently Achim is working on that …